test/Microsoft.TemplateEngine.Authoring.CLI.IntegrationTests/VerifyCommandTests.cs

```diff
@@ -0,0 +1,133 @@
+            File.Exists(Path.Combine(expectationsDir, templateDir, "Program.received.cs")).Should().BeTrue();
+            File.Exists(Path.Combine(expectationsDir, "std-streams", "stdout.received.txt")).Should().BeTrue();
+            File.Exists(Path.Combine(expectationsDir, "std-streams", "stderr.received.txt")).Should().BeTrue();
+            // .verified files are only created when diff tool is used - that is however turned off in CI
+            //File.Exists(Path.Combine(expectationsDir, "console.console.csproj.verified.csproj")).Should().BeTrue();
+            //File.Exists(Path.Combine(expectationsDir, "console.Program.cs.verified.cs")).Should().BeTrue();
+            //File.Exists(Path.Combine(expectationsDir, "console.StdOut.verified.txt")).Should().BeTrue();
+            //File.Exists(Path.Combine(expectationsDir, "console.StdErr.verified.txt")).Should().BeTrue();
+            Directory.GetFiles(expectationsDir, "*", SearchOption.AllDirectories).Length.Should().Be(4);
+
+            // .verified files are only created when diff tool is used - that is however turned off in CI
+            //File.ReadAllText(Path.Combine(expectationsDir, "console.console.csproj.verified.csproj")).Should().BeEmpty();
+            //File.ReadAllText(Path.Combine(expectationsDir, "console.Program.cs.verified.cs")).Should().BeEmpty();
+            //File.ReadAllText(Path.Combine(expectationsDir, "console.StdOut.verified.txt")).Should().BeEmpty();
+            //File.ReadAllText(Path.Combine(expectationsDir, "console.StdErr.verified.txt")).Should().BeEmpty();
+            File.ReadAllText(Path.Combine(expectationsDir, templateDir, "console.received.csproj").UnixifyLineBreaks()).Should()
+                .BeEquivalentTo(File.ReadAllText(Path.Combine(workingDir, templateDir, "console.csproj")).UnixifyLineBreaks());
+            File.ReadAllText(Path.Combine(expectationsDir, templateDir, "Program.received.cs").UnixifyLineBreaks()).Should()
+                .BeEquivalentTo(File.ReadAllText(Path.Combine(workingDir, templateDir, "Program.cs")).UnixifyLineBreaks());
+
+            // Accept changes
+            File.Move(
+                Path.Combine(expectationsDir, templateDir, "console.received.csproj"),
+                Path.Combine(expectationsDir, templateDir, "console.verified.csproj"));
+            File.Move(
+                Path.Combine(expectationsDir, templateDir, "Program.received.cs"),
+                Path.Combine(expectationsDir, templateDir, "Program.verified.cs"));
+            File.Move(
+                Path.Combine(expectationsDir, "std-streams", "stdout.received.txt"),
+                Path.Combine(expectationsDir, "std-streams", "stdout.verified.txt"));
+            File.Move(
+                Path.Combine(expectationsDir, "std-streams", "stderr.received.txt"),
+                Path.Combine(expectationsDir, "std-streams", "stderr.verified.txt"));
```

Comment:
I think we should move to using `.received` and `.verified` on folder instead. 
https://github.com/VerifyTests/Verify/blob/main/docs/naming.md#usesplitmodeforuniquedirectory